moveit_ros/planning/kinematics_plugin_loader/src/kinematics_plugin_loader.cpp

```diff
@@ -287,78 +294,71 @@ robot_model::SolverAllocatorFn KinematicsPluginLoader::getLoaderFunction(const s
 
       if (default_solver_plugin_.empty())
       {
-        ROS_DEBUG_NAMED(LOGNAME, "Loading settings for kinematics solvers from the ROS param server ...");
-
-        // read data using ROS params
-        ros::NodeHandle nh("~");
+        RCLCPP_INFO(LOGGER_KINEMATICS_PLUGIN_LOADER,
+                    "Loading settings for kinematics solvers from the ROS param server ...");
 
+        auto ksolver_params = std::make_shared<rclcpp::SyncParametersClient>(node_);
```

Comment:
Could you remove the `SyncParametersClient` and just use `rclcpp::Node` everywhere? It's much simpler than using a class we don't really need.